### PR TITLE
Remove unused classes

### PR DESCRIPTION
**Pull request by jgoodell-gsa**
Ready to merge

### CODE REVIEW

File: ProcessCIW/ProcessCIW.csproj [hunk 1]
```diff
@@ -97,11 +91,15 @@
       <SubType>Designer</SubType>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="AppSettings.config" />
+    <None Include="AppSettings.config">
```
Comment (tsaunders-gsa) [inline ProcessCIW/ProcessCIW.csproj]: Why is this in the ProcessCIW.csproj @jgoodell-gsa

File: ProcessCIW/ProcessCIW.csproj [hunk 2]
```diff
@@ -97,11 +91,15 @@
...
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
     <None Include="AppSettings.example">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="ConnectionStrings.config" />
+    <None Include="ConnectionStrings.config">
```
Comment (tsaunders-gsa) [inline ProcessCIW/ProcessCIW.csproj]: Why is this in the ProcessCIW.csproj @jgoodell-gsa

File: ProcessCIW/ProcessCIW.csproj (hunk 1 above)
Comment (jgoodell-gsa) [inline ProcessCIW/ProcessCIW.csproj]: Copy to output directory was set to do not copy and would not run until I changed it to copy if newer

File: ProcessCIW/ProcessCIW.csproj (hunk 2 above)
Comment (jgoodell-gsa) [inline ProcessCIW/ProcessCIW.csproj]: Copy to output directory was set to do not copy and would not run until I changed it to copy if newer

**Pull request merged**